gtsam/inference/EliminateableFactorGraph.h

```diff
@@ -111,13 +112,13 @@ namespace gtsam {
      *  \code
      *  VariableIndex varIndex(graph); // Build variable index
      *  Data data = otherFunctionUsingVariableIndex(graph, varIndex); // Other code that uses variable index
-     *  boost::shared_ptr<GaussianBayesNet> result = graph.eliminateSequential(EliminateQR, varIndex, boost::none);
+     *  boost::shared_ptr<GaussianBayesNet> result = graph.eliminateSequential(EliminateQR, varIndex, std::nullopt);
```

Comment:
Why nullopt here rather than {}

Reply:
I think I did that make the documentation clear